src/exco/extractor/locator/built_in/below_of_locator.py

```diff
@@ -23,6 +24,13 @@ def locate(self, anchor_cell_location: CellLocation,
                         sheet_name=anchor_cell_location.sheet_name,
                         coordinate=util.shift_coord(cell.coordinate, (1, 0))
                     )
+                    if self.empty:
```

Comment:
Refactor this please this code is very bad why would you need to construct the object everytime then throw away
???